WalletWasabi.Tests/RegressionTests/WabiSabi/Startup.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using WalletWasabi.WabiSabi.Models.Serialization;
+
+namespace WalletWasabi.Tests.RegressionTests.WabiSabi
```

Comment:
why `RegressionTests`? to me that term means testing to ensure that regressions don't happen, i.e. something that was previously working stops working (in practice, usually added as part of fixing a regression that was missed by tests that were in place at the time that the regression was accidentally introduced)

(sorry, i missed this in the PR that introduced `RegressionTests/WabiSabi`)